tox.ini

```diff
@@ -33,7 +33,7 @@ testpaths = tests
 # Recompile all requirements .txt files using pip-compile.
 # Don't edit me - I'm deployed from a template.
 deps = pip-tools
-basepython = python3.9
+basepython = python3.8
```

Comment:
Considering that `tox` creates a virtual environment [before running the commands](https://tox.wiki/en/latest/user_guide.html#system-overview) it should run every test inside the 3.8

Reply:
Yes, but it won't be able to create a virtual environment for python 3.8 if python 3.8 isn't already installed.  It can't install missing versions of python.

That's why I was asking if python 3.8 is guaranteed to be installed, as the github workflow is only asking for python 3.9 to be installed.